spec/lib/facilities/ppms/responses/response_spec.rb

```diff
@@ -0,0 +1,259 @@
+        'CareSiteAddress' => '420 W JUBAL EARLY DRIVE STE 203 WINCHESTER,VA 22601',
+        'WorkHours' => nil,
+        'ProviderGender' => 'NotSpecified',
+        'ProviderNetwork' => 'Provider Agreement',
+        'NetworkId' => 8,
+        'ProviderAcceptingNewPatients' => false,
+        'ProviderPrimaryCare' => false,
+        'QualityRanking' => 0,
+        'ProviderIdentifier' => '1114172319',
+        'Latitude' => 39.16996,
+        'Longitude' => -78.18052
+      }
+    ]
+  end
+
+  let(:services_response_body) do
+    [
+      {
+        'Name' => 'Danzer,Hal  - Obstetrics & Gynecology    ',
+        'AffiliationName' => 'TriWest',
+        'ProviderName' => 'Danzer,Hal ',
+        'ProviderAgreementName' => nil,
+        'SpecialtyName' => 'Obstetrics & Gynecology   ',
+        'ProviderSpecialtyName' => nil,
+        'CareSiteLocationAddress' => 'SOUTHERN CALIF REPRODUCTIVE CTR MED GRP INC',
+        'Longitude' => -122.5,
+        'Latitude' => 45.8,
+        'CareSiteAddressStreet' => '123 test street',
+        'CareSiteAddressZipCode' => '11111',
+        'CareSiteAddressCity' => 'Testville',
+        'CareSiteAddressState' => 'VA',
+        'CareSitePhoneNumber' => '(888) 444-1234',
+        'OrganiztionGroupName' => nil,
+        'DescriptionOfService' => nil,
+        'Limitation' => nil
+      },
+      {
+        'Name' => 'Danzer,Hal  - Obstetrics & Gynecology - Reproductive Endocrinology  ',
+        'AffiliationName' => 'TriWest',
+        'ProviderName' => 'Danzer,Hal ',
+        'ProviderAgreementName' => nil,
+        'SpecialtyName' => 'Obstetrics & Gynecology - Reproductive Endocrinology ',
+        'ProviderSpecialtyName' => nil,
+        'CareSiteLocationAddress' => 'SOUTHERN CALIF REPRODUCTIVE CTR MED GRP INC',
+        'CareSitePhoneNumber' => nil,
+        'OrganiztionGroupName' => nil,
+        'DescriptionOfService' => nil,
+        'Limitation' => nil
+      }
+    ]
+  end
+
+  let(:specialties_response_body) do
+    [
+      {
+        'SpecialtyCode' => '101Y00000X', 'Name' => 'Counselor   ',
+        'Grouping' => 'Behavioral Health & Social Service Providers',
+        'Classification' => 'Counselor',
+        'Specialization' => nil,
+        'SpecialtyDescription' => "A provider who is trained and educated in the performance of behavior health services
+         through interpersonal communications and analysis.
+         Training and education at the specialty level usually requires a master’s
+         degree and clinical experience and supervision for licensure or certification."
+      },
+      {
+        'SpecialtyCode' => '101YA0400X',
+        'Name' => 'Counselor - Addiction (Substance Use Disorder) ',
+        'Grouping' => 'Behavioral Health & Social Service Providers',
+        'Classification' => 'Counselor',
+        'Specialization' => 'Addiction (Substance Use Disorder)', 'SpecialtyDescription' => 'Definition to come...'
+      }
+    ]
+  end
+
+  let(:response_body) do
```

Comment:
Is there a reason this isn't a file fixture?

Reply:
Good idea @johnpaulashenfelter, moving to fixtures!